packages/core/src/types-hoist/options.ts

```diff
@@ -26,7 +26,11 @@ export interface ClientOptions<TO extends BaseTransportOptions = BaseTransportOp
 
   /**
    * A flag enabling Sessions Tracking feature.
-   * By default, Sessions Tracking is enabled.
+   * By default, Session Tracking is enabled.
+   *
+   * @deprecated Setting the `autoSessionTracking` option is deprecated.
+   * To enable session tracking, it is recommended to unset `autoSessionTracking` and ensure that either, in browser environments the `browserSessionIntegration` is added, or in server environments the `httpIntegration` is added.
+   * To disable session tracking, it is recommended unset `autoSessionTracking` and to remove the `browserSessionIntegration` in browser environments, or configure the `httpIntegration` with the `trackIncomingRequestsAsSessions` option set to `false`.
```

Comment:
l: Is this last part about trackIncomingRequestsAsSessions specific to server environments?